wal_downloader.go

```diff
@@ -15,12 +15,12 @@ import (
 // WALDownloader represents a parallel downloader of WAL files from a replica client.
 //
 // It works on a per-index level so WAL files are always downloaded in their
-// entirety and are not segmented. WAL files are downloaded from minIndex to
+// entiretry and are not segmented. WAL files are downloaded from minIndex to
```

Comment:
The original (entirety) is correct.

Reply:
@mtlynch Sorry about that. Must have been going too fast.

Pushed the fix to put it back to the original(correct) `entirety`...

Best,
Ryan